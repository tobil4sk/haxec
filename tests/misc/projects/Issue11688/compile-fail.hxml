--main Main
--macro exclude('Foo')
--interp