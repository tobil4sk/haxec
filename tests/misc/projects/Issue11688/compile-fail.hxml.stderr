Field index for foo not found on prototype _Foo.Foo_Fields_